bin/varnishtest/tests/u00001.vtc

```diff
@@ -61,3 +61,37 @@ shell -err {
 	varnishadm -n ${v1_name} -p |
 	python3 -c 'import sys, json; print(json.load(sys.stdin))'
 }
+
+shell {
+	cat > cli_file.txt <<-EOF
+	status
+	foo
+	status
+	EOF
+}
+
+shell { varnishadm -n ${v1_name} < cli_file.txt }
+shell { test 2 -eq $(varnishadm -n ${v1_name} < cli_file.txt | grep "Child in state" | wc -l) }
+
+shell -exit 101 { varnishadm -e -n ${v1_name} < cli_file.txt }
+shell { test 1 -eq $(varnishadm -e -n ${v1_name} < cli_file.txt | grep "Child in state" | wc -l) }
+
+shell { cat cli_file.txt | varnishadm -n ${v1_name} }
+shell { test 2 -eq $(cat cli_file.txt | varnishadm -n ${v1_name} | grep "Child in state" | wc -l) }
+
+shell -exit 101 { cat cli_file.txt | varnishadm -e -n ${v1_name} }
+shell { test 1 -eq $(cat cli_file.txt | varnishadm -e -n ${v1_name} | grep "Child in state" | wc -l) }
```

Comment:
I think we can do something simpler:
- run `varnishadm <file` without and with `-e`
  - no need to pipe with `cat` 
- don't use `status` twice, for example `ping`
- inspect the output accordingly

Reply:
I tested both `varnishadm.. < file` and the `cat file | varnishadm..` versions on purpose, that is because, for some reason, the bug fixed in 9dfc726172b2f411bdd5fbae6d8187cd216f2384 only affected the `cat file | varnishadm..` version and not the other one, so I thought it wouldn't hurt having some coverage for both.

Reply:
I see, regular file vs pipe... Then let's keep both, but the other suggestions apply.

Reply:
Fixed: Replaced the second status with vcl.list (which is easier to check compared to ping response that gives different output on each call), and added output inspection with diff